### DISCUSSION TITLE
Autoscale Images

### DISCUSSION THREAD

**Discussion by afischi** [inventwo/ioBroker.vis-inventwo#337] [Ideas]
Hi

also erst mal Mega Lob. Endlich mal ein Button der schon mal so funktioniert wie ich das von einer Mediensteuerung erwarte !!!

Jetzt kann es sein ich bin zu blöd, oder das geht nicht. 
Aber wenn ich da ein Bild auswähle für true und für false. Kann man nicht eine Option reinmachen, Scaliere das auf die Größe des Buttons?
Oder gibt es einen Trick wie ich das hinzugefügte Bild oder nennt es Icon so scaliere daß es Button füllend ist ?
Weil so sieht es einfach blöd aus.

![image](https://user-images.githubusercontent.com/48256727/142998642-56f9ecde-68b6-4896-9526-4f36b103aa45.png)

Gruß
 Afischi

**Comment by Haifeif** [comment]
Das unterstütze ich: Die "großen" Systeme wie CRESTRON oder RTI glänzen genau mit solchen Dingen! Es wäre Weltklasse, wenn es dafür eine Lösung gäbe!

**Comment by jkvarel** [comment]
Hab irgendwie jetzt erst bemerkt, dass es einen Tab "Discussions" gibt...

Falls das Problem noch besteht, bei Bildgröße kann auch Prozent verwendet werden 
![image](https://user-images.githubusercontent.com/35769746/154298060-dbfcb29c-aeb5-4583-9630-db1326727182.png)

**Reply by afischi** [reply]
Haha kein Problem, ich muß mich hier auch erst zurechtfinden.
Ich habe das inzwischen getestet Danke geht :)

Ich hab aber das Gefühl wenn das Bild das man einbindet nur 40x40 px ist dann kann das auch maximal so dargestellt werden. Wenn mein Button auf 100x100px ziehe dann bleibt das image bei 40x40 stehen und scaliert nicht weiter. Muss wohl irgend eine Limitierung sein von irgendwas. Aber nicht schlimm. Ich habe auch nicht eure Icons genommen sondern was anderes als Bild.

Aber Super Job gemacht!!!

DANKE !!!
